### PR TITLE
v1.7.6 updates

### PR DESCRIPTION
**Pull request by fkneeland-figure**
## Description

closes: #XXXX

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/provenance-io/provenance/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote unit and integration [tests](https://github.com/provenance-io/provenance/blob/main/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

**Comment by fkneeland-figure** [comment]
This addresses the missing pieces in: https://github.com/provenance-io/provenance/pull/571/files

File: RELEASE_CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+## [v1.7.6](https://github.com/provenance-io/provenance/releases/tag/v1.7.6) - 2021-12-15
+
+### Improvements
+
+* Upgrade Rosetta to v0.7.2 [#560](https://github.com/provenance-io/provenance/issues/560)
```
Comment (iramiller) [inline RELEASE_CHANGELOG.md:5]:
```suggestion
* Upgrade Rosetta to v0.7.2 to resolve issue with coins minted after genesis. [#560](https://github.com/provenance-io/provenance/issues/560)
```

**Review by iramiller — CHANGES_REQUESTED** [review]
Overall looks good ... feels like we should maybe add a little more detail in the release change log indicating that this is for the Rosetta fix ... possibly mention that this version is only required for users that are running a rosetta server instance.

**Comment by fkneeland-figure** [comment]
@iramiller I updated the release notes.  Is that what you were thinking?

**Review by dwedul-figure — APPROVED** [review]
LGTM

**Review by iramiller — APPROVED** [review]

**Review by dwedul-figure — APPROVED** [review]
LGTM

**Pull request merged**